doc/build-unix.md

```diff
@@ -61,13 +61,17 @@ Versions used in this release:
 
 Dependency Build Instructions: Ubuntu & Debian
 ----------------------------------------------
-sudo apt-get install build-essential
-sudo apt-get install libssl-dev
-sudo apt-get install libdb4.8-dev
-sudo apt-get install libdb4.8++-dev
- Boost 1.40+: sudo apt-get install libboost-all-dev
- or Boost 1.37: sudo apt-get install libboost1.37-dev
-sudo apt-get install libqrencode-dev
+- sudo apt-get install build-essential
+- sudo apt-get install libssl-dev
+- sudo apt-get install libdb4.8-dev
+- sudo apt-get install libdb4.8++-dev
```

Comment:
I mean... I suppose we could, I know bitcoin recommends 4.8 for portability purposes but it's also particularly out of date.

If we're going to recommend a newer version we really should probably be compiling the gitian builds in the same version so that wallets can be opened by either binary.